ui-test/README.md

```
@@ -1,129 +1,366 @@
 
-Set the environment variables as follows:
+---
+
+## Project Structure
 
-```bash
-export INJIWEB_GOOGLE_REFRESH_TOKEN=<<refresh_token>>
-export INJIWEB_GOOGLE_CLIENT_ID=<<google_client_id>>
-export INJIWEB_GOOGLE_CLIENT_SECRET=<<client_secret>>
-export TEST_URL=<<url>>
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)